Rebrand

JasonStoltz opened the pull request:
- Updated “Swiftype” references with “Elastic” in the readme and in the code.
- Renamed the published package to `elastic-enterprise-search`
- Changed versioning to be pre-release again, 0.1.0, since we are not yet in GA, just like we are doing for App Search.
- Updated copyright and license. Project is now Apache 2.0.
- Sending the analytics headers: X-Swiftype-Client and X-Swiftype-Client-Version
- Enabled CircleCI.

yakhinvadim commented:
Starting review

yakhinvadim commented on doc/api/index.rst:
```diff
@@ -3,5 +3,4 @@ Swiftype Enterprise Client API
 
```
Is the first line in file can be changed here?

yakhinvadim commented on README.md:
```diff
@@ -38,15 +40,15 @@ Retrieve your access token and a content source key after creating your content
-      'url': 'https://github.com/swiftype/swiftype-enterprise-python',
-      'title': 'Swiftype Enterprise Python Github',
+      'url': 'https://github.com/elastic/enterprise-search-python',
+      'title': 'Elastic App Search Official Python Client',
```
Enterprise*

yakhinvadim commented on doc/api/swiftype_enterprise_client.rst:
```diff
@@ -1,7 +1,7 @@
-:class:`~swiftype_enterprise.SwiftypeEnterpriseClient`
+:class:`~elastic_enterprise_search.SwiftypeEnterpriseClient`
```
We can't change this `SwiftypeEnterpriseClient`? Same for line 6.

yakhinvadim commented on doc/index.rst:
```diff
@@ -17,7 +17,7 @@ the docs are on `ReadTheDocs <https://swiftype_enterprise.readthedocs.io/>`_.
 
```
There are a lot of swiftype references above this line. Do we want to keep them?

JasonStoltz replied:
@yakhinvadim Whoops. I meant to circle back and address those doc files. Sorry about that, will tag you again when they're updated.

JasonStoltz replied:
@yakhinvadim I ended up deleting these docs. I don't really know if they were generated or manually maintained. However, none of our other Python client libraries contain docs like these, and I'd like to not maintain them going forward.

JasonStoltz commented:
@yakhinvadim Ready for a second look.

JasonStoltz commented:
You've gone above and beyond :)

JasonStoltz commented:
- Updated “Swiftype” references with “Elastic” in the readme and in the code.
- Renamed the published package to `elastic-enterprise-search`
- Changed versioning to be pre-release again, 0.1.0, since we are not yet in GA, just like we are doing for App Search.
- Updated copyright and license. Project is now Apache 2.0.
- Sending the analytics headers: X-Swiftype-Client and X-Swiftype-Client-Version
- Enabled CircleCI.
